### PR TITLE
Indicate pair programming on exercise

### PR DESCRIPTION
**Pull request by zacheliason**

### CODE REVIEW

File: front_end/html/navbar_exercise.html [hunk 1]
```diff
@@ -16,6 +16,9 @@
                                     <img src="/static/unchecked.png" alt="Not completed" class="navbar-img"/>
                                 {% end %}
                                 {{ exercise[1]["title"] }}
+                                {% if exercise[1]["enable_pair_programming"] %}
```
Comment (srp33) [inline front_end/html/navbar_exercise.html:19]: I mentioned this in an email, but please add a space after the title and before the P.

Comment (srp33) [reply front_end/html/navbar_exercise.html:19]: Please also make sure the text won't wrap if we have a longer title (or the screen is narrow).

**Review by srp33 — CHANGES_REQUESTED** [review]

File: front_end/html/navbar_exercise.html (hunk 1 above)
Comment (zacheliason) [reply front_end/html/navbar_exercise.html:19]: Just to clarify, this is how the text currently wraps under worst case conditions. (this is the longest possible title and the narrowest screen I could make) Do you want me to make the text flow off the screen to the right?
image

Comment (srp33) [reply front_end/html/navbar_exercise.html:19]: This looks good. I like that it wraps. Not to be a pain, but it appears that the text is aligned vertically with the bottom of the status icon to the left of the text. Is there a way to center it vertically?

Comment (zacheliason) [reply front_end/html/navbar_exercise.html:19]: Yes, I just fixed it!

**Pull request merged**